src/srvctl/main.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (C) 2022, KNS Group LLC (YADRO).
+ */
+#include "config.h"
+
+#include "srvctl.hpp"
+#include "utils/confirm.hpp"
+
+#include <fmt/printf.h>
+
+#include <CLI/CLI.hpp>
+
+#include <stdexcept>
+
+static void setState(sdbusplus::bus::bus& bus, const std::string& service,
+                     bool state, bool interactive)
+{
+    auto it = srvctl::serviceDefinitions.find(service);
+    if (it == srvctl::serviceDefinitions.end())
+    {
+        throw std::runtime_error(
+            fmt::format("Service {} is invalid or unsupported", service));
```

Comment:
Since this is a new utility, could you please design it from the start for i18n using `gettext()` ?

Reply:
I have doubts about this idea.
In BMC all i18 settings are specially truncated to keep the minimal rootfs size.
Are you sure that we really need it?

Reply:
We don't need "all" i18n, but CLI messages will eventually need to be translated to at least Russian. I think it's better to take that into consideration at the inception of the new component.